src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
```

Comment:
you import wrong settings of font, so you have some problems on tests.
check font style for modificator's in your code
```suggestion
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
```